packages/website/components/account/paymentMethodCard/paymentMethodCard.js

```diff
@@ -0,0 +1,37 @@
+import { faCcVisa } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
I think this might require a license for commercial projects. Is there something else we can use? If not obvious, let's just not have an icon for now?

Reply:
the brand icon pack is free, but if we have a definitive list I could make these myself.